### DISCUSSION TITLE
Feedback on the Lists beta feature.

### DISCUSSION THREAD

**Discussion by webbertakken** [community/community#29270] [Other Feature Feedback, Questions, & Ideas]
Having categorised my 300 stars in categories, using lists, I noticed the following things.

#### First impression
- First impression is really good. It works intuitively and having slightly more granular control over "My stars" is a very welcome change.
- Being able to add utf-8 icons in the category name is very helpful.

#### Some notes
- With categories, you no longer are incentivised to star a repository. It would be cool if starring is implicit when adding a repo to a list.
- There is no easy way to find out which starred repositories are not yet categorised. I solved this by going through all of them in one go and moving everything that didn't have a clear category to a group called "❓Uncategorised"
- Given the following list, I wouldn't want the two categories with only 1 item in it to show up at the top of the list. 
![image](https://user-images.githubusercontent.com/20756439/184350624-6e9e3914-4798-433c-8ce5-b67903de7073.png)
- There currently is no way to sort your lists by number of items. Either pinning or sorting by number of items would be nice. Both would be great.
- Edit: The feedback button links to a page that doesn't show results (causing people to duplicate post, like I probably did just now) 
![image](https://user-images.githubusercontent.com/20756439/184351316-2c169762-efe6-44f7-b046-4ab6749efc7a.png)

#### What I'd love to see
- Adding the top 3~12 lists to your profile page, much like pinned repositories.
- Quickly finding out which items are in multiple lists

**Comment by mubeeum** [comment]
Yes, It will be nice to have pin functionality in lists.

![image](https://user-images.githubusercontent.com/42890603/190211023-5842eb69-5bf1-433a-bf64-bcf507f9de1e.png)
